compiler/src/dotty/tools/dotc/ast/TreeInfo.scala

```diff
@@ -814,17 +814,31 @@ trait TypedTreeInfo extends TreeInfo[Type] { self: Trees.Instance[Type] =>
       case _ => false
     }
 
-  /** An extractor for closures, either contained in a block or standalone.
+  /** An extractor for closures, possibly typed, and possibly including the
+   *  definition of the ananonymous def.
```

Comment:
... the definition of an anonymous def or _the_ anonymous def?

Reply:
`the` def that corresponds to the closure node.

Reply:
```suggestion
   *  definition of the anonymous def.
```